webpack.config.js

```diff
@@ -45,7 +46,46 @@ module.exports = {
         new webpack.BannerPlugin({
             banner: "Copyright (c) 2020, Alexandra Rose Charity (reg. in England and Wales, #00279157)",
         }),
-        new OfflinePlugin()
+        new OfflinePlugin(),
+        new WebpackPwaManifest({
+            name : 'Rosie - Rose Voucher Records & Reimbursement',
+            short_name : 'Rosie',
+            display_name : 'Rosie - Rose Voucher Records & Reimbursement',
+            description : 'Record your Rose Vouchers.',
+            lang: 'en',
+            dir: 'ltr',
+            theme_color : '#a74e94',
+            background_color : '#fff',
+            orientation : 'natural',
+            scope : '/',
+            icons : [
+                {
+                    src: path.resolve('src/assets/launcher-48x48.png'),
+                    size: '48x48',
+                },
+                {
+                    src: path.resolve('src/assets/launcher-96x96.png'),
+                    size: '96x96',
+                },
+                {
+                    src: path.resolve('src/assets/launcher-144x144.png'),
+                    size: '144x144',
+                },
+                {
+                    src: path.resolve('src/assets/launcher-192x192.png'),
+                    size: '192x192',
+                },
+                {
+                    src: path.resolve('src/assets/icon.svg'),
+                    size: '193x193',
+                },
+                {
+                    src: path.resolve('src/assets/launcher-512x512.png'),
+                    size: '512x512',
```

Comment:
I wonder if we need more/new/reformatted Icons for the PWA standard - both play and istore have changed their standards, has the PWA changed too?

Reply:
Yes - we probably do, but I don't think it's a priority. This plugin seems to be able to generate sizes from one icon and an array of sizes... but I thought I'd leave that for a day in the distant future.

I'll note an AC on the card for @jermyk to test the icon for all supported devices.